src/tutorials/camera_capture_basics.md

```diff
@@ -0,0 +1,238 @@
+| **Free Control** | Should be left unchecked. |
+
+![!Camera Properties](../img/CameraCapture/Basics/basics_camera_properties.png){: .center loading="lazy" }
+
+## Create CameraCaptureClient Script
+
+To be able to create camera captures, a script will be required that will use the [**Capture**](../api/camera.md) function to create a capture and return it so you can use it.
+
+Create a new script called `CameraCaptureClient` and place it inside the **Client Context** folder in the **Hierarchy**.
+
+![!Script](../img/CameraCapture/Basics/basics_script.png){: .center loading="lazy" }
+
+### Create Custom Properties
+
+The **CameraCaptureScript** needs a reference to the **UI Image** and the **Camera** you created earlier.
+
+1. Add the **UI Image** in the **Hierarchy** to the **CameraCaptureScript** as a **Custom Property**.
+2. Add the **Camera** in the **Hierarchy** to the **CameraCaptureScript** as a **Custom Property**.
+
+![!Properties](../img/CameraCapture/Basics/basics_script_properties.png){: .center loading="lazy" }
+
+### Create Property References
+
+Open up the **CameraCaptureScript** and create references to the **UI Image** and **Camera**. The reason you need both of these is because you will use the `CAMERA` to create the camera capture, and the `UIMAGE` to display the camera capture in the UI.
+
+```lua
+local UIIMAGE = script:GetCustomProperty("UIImage"):WaitForObject()
+local CAMERA = script:GetCustomProperty("Camera"):WaitForObject()
+```
+
+## Create Camera Capture
+
+To create a camera capture, you can call the `Capture` function and pass the [resolution size](../api/cameracapture.md) as the first argument. You can create any combination of camera captures, but there is a memory budget which is explained in a later section. If you do reach the limit, further attempts to create camera capture instances will fail. You should then consider reusing or releasing existing captures.
+
+The camera capture is stored in the `capture` variable and passed into the `SetCameraCapture` function. The `UIImage` will be set to the camera capture that will show in the UI. It is good practice to make sure the capture exists to prevent errors, which can be done by using an `if` statement.
+
+```lua
+local capture = CAMERA:Capture(CameraCaptureResolution.LARGE)
+
+if capture then
+    UIIMAGE:SetCameraCapture(capture)
+end
+```
+
+### The CameraCaptureClient Script
+
+??? "CameraCaptureClient"
+    ```lua
+    local UIIMAGE = script:GetCustomProperty("UIImage"):WaitForObject()
+    local CAMERA = script:GetCustomProperty("Camera"):WaitForObject()
+
+    local capture = CAMERA:Capture(CameraCaptureResolution.LARGE)
+
+    if capture then
+        UIIMAGE:SetCameraCapture(capture)
+    end
+    ```
+
+### Test the Game
+
+Test the game and see if the UI image displays the camera capture.
+
+![!First Capture](../img/CameraCapture/Basics/basics_first_capture.png){: .center loading="lazy" }
+
+### Challenge
+
```

Comment:
Have you used challenges in other tutorials? Not saying it's wrong just interested to see if it has been done before.

Reply:
Sort of. More of a list in the summary, not as it's own section. 

I am considering removing it though, I think I want to expand on the previous section a little.